content-node/docker-compose/setup.sh

```diff
@@ -141,6 +141,23 @@ get_validated_input() {
     done
 }
 
+# Check if .env file exists and prompt for confirmation
+if [ -f ".env" ]; then
+    log_warning "An .env file already exists!"
+    read -p "Do you want to overwrite it? (y/n): " overwrite
```

Comment:
can we make the text here like "Would you like to go through the .env setup process again?"